### PR TITLE
Added docs server script, dark mode & copybutton for docs

### PR DESCRIPTION
**Pull request by OmkarPh**
Possible additions to the scancode-toolkit documentation:

A key terminologies section 
  * PURL
  * Packages
  * License detection
  (Already available - https://scancode-toolkit.readthedocs.io/en/latest/reference/license-detection-reference.html)
      * License keys & Policies
      * License clues
  * Dependencies
      * Scopes
      * Data source IDs
      * Runtime / Resolved / Optional deps

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁
* [x] Looked for possible updates in documentation and added updates if applicable
* [ ] Updated CHANGELOG.rst

### CODE REVIEW

File: docs/source/tutorials/how_to_visualize_scan_results.rst [hunk 1]
```diff
@@ -3,109 +3,15 @@
...
-directory distributed by default with Scancode, and visualize the outputs through
-Scancode Workbench.
+To help visualize the scans, we have a dedicated tool  `Scancode workbench <https://github.com/nexb/scancode-workbench/>`_  which is a desktop application that allows you to visualize and explore the results of one or more scans. It is a cross-platform application that runs on Windows, Mac OS X and Linux. It is built using the Electron framework and is built using Electron, Typescript & React
+
+Detailed Installation and Usage guide can be found here -  :ref:`scancode-workbench:getting-started`
 
 .. WARNING::
 
-    This tutorial uses the 3.1.1 version of Scancode Toolkit, and Scancode Workbench 3.1.0 (This
+    This tutorial uses the 32.x version of Scancode Toolkit, and Scancode Workbench 4.0-beta (This
```
Comment (AyanSinhaMahapatra) [inline docs/source/tutorials/how_to_visualize_scan_results.rst]: This can be `4.0.x` as we're almost there

**Review by AyanSinhaMahapatra — APPROVED** [review]
LGTM!
Thanks @OmkarPh 
Please also add the relevant changes at https://github.com/nexB/skeleton

File: docs/source/tutorials/how_to_visualize_scan_results.rst (hunk 1 above)
Comment (OmkarPh) [reply docs/source/tutorials/how_to_visualize_scan_results.rst]: Sure, I've updated it

**Comment by AyanSinhaMahapatra** [comment]
Thanks @OmkarPh , merging!

**Pull request merged**